specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/cdnwebapplicationfirewall.json

```diff
@@ -696,7 +703,11 @@
           "type": "array",
           "items": {
             "$ref": "#/definitions/ManagedRuleSet"
-          }
+          },
+          "x-ms-identifiers": [
+            "ruleSetType",
```

Comment:
Same question here. Are we trying to be as descriptive as possible?

Reply:
Yes, we are making sure that the identifier is really unique for that object. Usually one property is not enough.





Yes, we are making sure that the identifier is really unique for that object. Usually one property is not enough.